### ISSUE TITLE
disallow duplicates at the same time

### ISSUE THREAD

**Issue by HoWRiM** [aldabil21/react-scheduler#124]
Hi,
Thanks for this awesome package.

Is there any way to disallow duplicates at the same time?
I looked hard at the documentation but couldn't find it.

**Comment by aldabil21** [comment]
Hi

Although there is no direct option/prop to make it possible, it is doable by doing the following:

1. Use `onConfirm` and make sure you validate the add/edit functionality, if there is an event in the same slot, the intercept and return false. Another option is, you can use `customEditor` to make your own form/fields and also validate before save. However, validation should be in your backend to prevent saving 2 events in the same slot.
2. Use `onEventDrop` top intercept dropping event and validate if the slot already has an event, then return void/false.

Feel free to re-open if you still have an issue.

**Comment by HoWRiM** [comment]
I'm writing a comment after a long time. 
I applied this library to my personal project, a place reservation system. Since it involves reserving a place, it was crucial to prevent event overlaps. 
I struggled for several days and found it difficult to customize the library. 
In the end, I checked the values for event creation, modification, and drag using the time-table status feedback. 
Finally, I prevented overlaps by checking with a unique index in MongoDB.
